src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -89,6 +94,12 @@ module sonic-device_metadata {
                         pattern "ToRRouter|LeafRouter|SpineChassisFrontendRouter|ChassisBackendRouter|ASIC";
                     }
                 }
+                leaf frr_mgmt_framework_config {
+                    type boolean;
+                    description "FRR configurations are handled by sonic-frr-mgmt-framework module when set to true, 
```

Comment:
should this be not handled by "docker_routing_config_mode" in switch METADATA block ?

Reply:
No, we already have a backend based on frr_mgmt_framework_config in the community to enable frrcfgd (generic config DB tables for both BGP & route-map faetures), this is just the SONiC YANG model to match the backend design.